website/pages/docs/plugins/sources/gcp/tables/[table].mdx

```diff
@@ -0,0 +1,25 @@
+{/* <!-- vale off --> */}
```

Comment:
I think that this code should somehow be extracted and called only with `gcp` param (as the table gen is repeated across all source plugins).

Reply:
I think https://github.com/cloudquery/cloudquery/pull/8672/commits/e63ff01d45778cf8287ba91c04e4563fbe3e3ab4 is the best we can do